rct229/rulesets/ashrae9012019/section6/section6rule9.py

```diff
@@ -187,6 +188,16 @@ def rule_check(self, context, calc_vals=None, data=None):
                         total_hours_matched = calc_vals["total_hours_matched"]
                         return total_hours_matched == total_hours_compared
 
+                    def rule_check(self, context, calc_vals=None, data=None):
+                        total_hours_compared = calc_vals["total_hours_compared"]
+                        total_hours_matched = calc_vals["total_hours_matched"]
+                        return total_hours_matched == total_hours_compared
+
+                    def is_tolerance_fail(self, context, calc_vals=None, data=None):
+                        total_hours_compared = calc_vals["total_hours_compared"]
+                        total_hours_matched = calc_vals["total_hours_matched"]
+                        return std_equal(total_hours_matched, total_hours_compared)
```

Comment:
1. put the `is_tolerance_fail` after `get_fail_msg` because the function is called after `get_fail_msg`.
2. We do not need tolerance check for integers.